docs/workflow/testing/coreclr/unix-test-instructions.md

```diff
@@ -1,49 +1,57 @@
-Building and running tests on Linux, OS X, and FreeBSD
+Building and running tests on Linux, macOS, and FreeBSD
 ======================================================
 
 CoreCLR tests
 -------------
 
 ## Building
 
-Build CoreCLR on [Unix](https://github.com/dotnet/coreclr/blob/master/Documentation/building/linux-instructions.md).
+Build CoreCLR on [Unix](../../building/coreclr/linux-instructions.md).
 
 ## Building the Tests
 
-DotNet is required to build the tests, this can be done on any platform then copied over if the arch or os does not support DotNet. If DotNet is not supported, [CoreFX](https://github.com/dotnet/corefx/blob/master/Documentation/building/unix-instructions.md) is also required to be built.
+Dotnet CLI is required to build the tests. This can be done on any platform then copied over if the architecture or OS does not support Dotnet.
 
 To build the tests on Unix:
 
-> `./src/coreclr/build-test.sh`
+```sh
+./src/coreclr/build-test.sh
+```
 
 Please note that this builds the Priority 0 tests. To build priority 1:
 
-> `src/coreclr/build-test.sh -priority 1`
+```sh
+./src/coreclr/build-test.sh -priority 1
+```
 
 ## Building Individual Tests
 
-During development there are many instances where building an individual test is fast and necessary. All of the necessary tools to build are under `coreclr`. It is possible to use `runtime/.dotnet/dotnet msbuild` as you would normally use MSBuild with a few caveats.
+During development there are many instances where building an individual test is fast and necessary. All of the necessary tools to build are under `coreclr`. It is possible to use `~/runtime/dotnet msbuild` as you would normally use MSBuild with a few caveats.
 
 **!! Note !! -- Passing /p:__BuildOs=[OSX|Linux] is required.**
 
 ## Building an Individual Test
 
->`/path/to/runtime/.dotnet/dotnet msbuild src/coreclr/tests/src/path-to-proj-file /p:__BuildOS=<BuildOS> /p:__BuildType=<BuildType>`
+```
+/path/to/runtime/dotnet msbuild src/coreclr/tests/src/path-to-proj-file /p:__BuildOS=<BuildOS> /p:__BuildType=<BuildType>
```

Comment:
FYI @ViktorHofer  who is updating these properties to be consistent across partitions... this will need to be updated as part of your change.